jitsi/provision

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+for i in "$@"
+do
+case $i in
+    --instance-type=*)
+    INSTANCE_TYPE="${i#*=}"
+    shift
+    ;;
+    --ssl-certificate=*)
+    SSL_CERTIFICATE="${i#*=}"
+    shift
+    ;;
+    --public-domain=*)
+    PUBLIC_DOMAIN="${i#*=}"
+    shift
+    ;;
+    --include-nginx)
+    INCLUDE_NGINX=YES
+    shift
+    ;;
+esac
+done
+
+packer build \
+  -var 'instance_type=${INSTANCE_TYPE}' \
+  -var 'ssl_certificate=${SSL_CERTIFICATE}' \
+  -var 'public_domain=${PUBLIC_DOMAIN}' \
+  -var 'include_nginx=${INCLUDE_NGINX}'\
+  example.json
```

Comment:
Consider making `jitsi/provision` to take different VM provider as argument and choose different `aws-ebs.json` or `digital-ocean.json` but sticking with AWS only for now.